utilities/include/vec4.hpp

```diff
@@ -7,7 +7,6 @@
 #include <type_traits> // for std::is_constant_evaluated
 
 #include <xsimd/xsimd.hpp>
-
```

Comment:
Why did you remove this blank line? It exists to provide a little separation between a library include and our own project's include.